packages/core/src/auth/utils.ts

```diff
@@ -577,10 +577,12 @@ export class AuthNode implements TreeNode<Auth> {
                 command: showConnectionsPageCommand,
                 arguments: [placeholder, source],
             }
-
+            if (DevSettings.instance.isNewLoginUxEnabled()) {
+                void vscode.commands.executeCommand('setContext', 'aws.explorer.showAuthView', true)
+            }
             return item
         }
-
+        void vscode.commands.executeCommand('setContext', 'aws.explorer.showAuthView', false)
```

Comment:
why not `await` this? if `setContext` is doing something async, we want synchronous behavior, surely?